Cecilifier.Core/AST/ExpressionVisitor.cs

```diff
@@ -61,6 +61,17 @@ static ExpressionVisitor()
             operatorHandlers[SyntaxKind.LessThanToken] = (ctx, ilVar, left, right) => WriteCecilExpression(ctx, $"{ilVar}.Append({ilVar}.Create({OpCodes.Clt.ConstantName()}));");
             operatorHandlers[SyntaxKind.MinusToken] = (ctx, ilVar, left, right) => WriteCecilExpression(ctx, $"{ilVar}.Append({ilVar}.Create({OpCodes.Sub.ConstantName()}));");
             operatorHandlers[SyntaxKind.AsteriskToken] = (ctx, ilVar, left, right) => WriteCecilExpression(ctx, $"{ilVar}.Append({ilVar}.Create({OpCodes.Mul.ConstantName()}));");
+
+            // Bitwise Operators
```

Comment:
Please,

1. Add tests for each operator (you can find inspiration in this commit 70661c6c8b4cad2bb2dbc9e34fdb9135690c64a9)
1. Ideally the test for the bitwise operators should cover all integral types combinations (char x char, char x byte, char x int,  char x long, char x ulong, char uint, , byte x int, byte x uint, ...) (if there are missing conversions, please add an issue for each of the missing conversions, mark the test as ignored adding a reference to the issue)
1. Update [CHANGELOG.md](https://github.com/adrianoc/cecilifier/blob/main/CHANGELOG.md) to reflect the changes